### DISCUSSION TITLE
Ideas para logo

### DISCUSSION THREAD

**Discussion by claubott** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#166] [Ideas]
Los colores pueden cambiar, pero aquí van un par de ideas:
![imagen](https://user-images.githubusercontent.com/37263110/234446902-59671736-8c3a-4ada-8bd0-7494a5c8a5c9.png)
![imagen](https://user-images.githubusercontent.com/37263110/234447091-2b38d030-e71c-47e1-a64a-624194b7ff33.png)

**Comment by NataliaAlvarezIspc** [comment]
![flc_design20230426142680](https://user-images.githubusercontent.com/106571110/234465291-93303597-314d-4325-90c3-027825968ab7.png)

**Comment by NataliaAlvarezIspc** [comment]
Me gusta el primero de Clau, en gama de naranja puede quedar bueno.

**Comment by BraianTroncoso** [comment]
![envaldeLogo-Svg](https://user-images.githubusercontent.com/95662710/234491101-4bfd6711-6ff1-498b-bb94-5eee06d2b5d8.png)
Hice todo este logo por mi cuenta. Obviando que es de una heladeria agregé un helado como también su nombre.
A demás quise agregar algo con respecto al nombre que es "ENBALDE" ya que hace hincapié a "EMBALSE", le agregué un lago de fondo con un poco de sierras.

**Reply by BraianTroncoso** [reply]
Esta sin fondo como si fuera un SVG, para trabajarlo mas comódo en el sitio web. Tambien lo tengo en con un fondo neutral. Acá también comparto el otro:
![envaldeLogo](https://user-images.githubusercontent.com/95662710/234491407-bae8117d-7bd4-4ef4-aa33-49da88fefc64.png)
